src/error.h

```diff
@@ -13,6 +13,8 @@
 #define GAP_ERROR_H
 
 #include "system.h"
+#include "integer.h"
```

Comment:
Why is integer.h included?

Reply:
for some reason I thought I needed it for INT_INTOBJ, which obviously isn't true. removed.